src/lib/y2ntp_client/widgets/pool_widgets.rb

```diff
@@ -0,0 +1,114 @@
+require "yast"
+
+require "cwm/widget"
+
+Yast.import "NtpClient"
+
+module Y2NtpClient
+  module Widgets
+    # Input field with address
+    class PoolAddress < CWM::InputField
+      attr_reader :address
+
+      def initialize(initial_value)
+        textdomain "ntp-client"
+        @address = initial_value
+      end
+
+      def label
+        _("A&ddress")
+      end
+
+      def init
+        self.value = @address
+      end
+
+      def validate
+        # TODO: validate address and also that it is not yet used
+        true
+      end
+
+      def store
+        @address = value
+      end
+    end
+
+    # Button that tests if server is reachable
+    class TestButton < CWM::PushButton
+      def initialize(address_widget)
+        textdomain "ntp-client"
+        @address_widget = address_widget
+      end
+
+      def label
+        _("Test")
+      end
+
+      def handle
+        Yast::NtpClient.TestNtpServer(@address_widget.value, :result_popup)
+
+        nil
+      end
+    end
+
+    # Enable iburst option
+    class Iburst < CWM::CheckBox
+      def initialize(options)
+        textdomain "ntp-client"
+        @options = options
+      end
+
+      def label
+        _("Quick Initial Sync")
+      end
+
+      def init
+        self.value = @options.key?("iburst")
+      end
+
+      def store
+        if value
+          @options["iburst"] = nil
+        else
+          @options.delete("iburst")
+        end
+      end
+
+      def help
+        _("<b>Quick Initial Sync</b> specifies if option iburst is used. This option during " \
+          "initialization send four poll requests with two seconds interval. Useful to quick " \
+          "synchronization during start of machine.")
+      end
+    end
+
+    # Enable offline option
+    class Offline < CWM::CheckBox
+      def initialize(options)
+        textdomain "ntp-client"
+        @options = options
+      end
+
+      def label
+        _("Start Offline")
+      end
+
+      def init
+        self.value = @options.key?("offline")
+      end
+
+      def store
+        if value
+          @options["offline"] = nil
+        else
+          @options.delete("offline")
+        end
+      end
+
+      def help
+        _("<b>Start Offline</b> specifies if option offline is used. This option skip this " \
+          "server during start. It is useful for machine which start without network, because " \
+          "it speed up boot and sync when machine is connected to network.")
```

Comment:
```rb
_("<b>Start Offline</b> specifies whether the 'offline' option is used. This option skips this " \
"server during the start-up. It is useful for a machine which starts without the network, because " \
"it speeds up the boot, and synchronizes when the machine gets connected to the network.")
```